yogstation/code/modules/antagonists/traitor/datum_mindslave.dm

```diff
@@ -11,4 +11,19 @@
 	name = "Mindslave Objective"
 	martyr_compatible = TRUE
 	completed = TRUE
-	explanation_text = "Serve your master no matter what!"
\ No newline at end of file
+	explanation_text = "Serve your master no matter what!"
+
+/datum/antagonist/greytide
+	name = "Greytider"
+	antagpanel_category = "Mindslave"
+	job_rank = ROLE_TRAITOR
+	show_in_antagpanel = FALSE
+	antag_moodlet = /datum/mood_event/focused
+	can_hijack = FALSE
```

Comment:
I'd say remove this line so that they don't count against traitor hijacks

Reply:
0  is HIJACK_NEUTRAL which mean ignored for purposes of hijack

Reply:
Well this isn't a boolean anyways, just remove the line so it can go back to the default which is HIJACK_NEUTRAL